README.md

```diff
@@ -67,6 +67,48 @@ where:
 
 _This method calls `group.add` and `group.addAuthors` for you (adding you)_
 
+### `ssb.tribes.application.create(groupdId, recps, opts, cb)`
+
+Creates a tribe application message directed to the administrators of a private-group.
+
+where:
+- `opts` *Object*:
+  - `text` *String*: A text to be viewed by the Kaitiakis of a group
+
+### `ssb.tribes.application.get(applicationId, cb)`
+
+Returns the current state of a tribe application. Ex.:
+
+```
+{
+  root: '%CXVDe5AoPVf83CoHYBpfplpzTU/YYkN56yge1nBG9wE=.sha256',
+  applicantId: '@35wu1YDBx9NPsUXpe7bYmHb7BQFEfn2ZFh0DZ6OipA0=.ed25519',
+  groupId: '%A9OUzXtv7BhaAfSMqBzOO6JC8kvwmZWGVxHDAlM+/so=.cloaked',
+  recps:
+   [ '@CQi7RZDHLHalHErknddXIczj6FulnAdbYfULVSXTbns=.ed25519',
+     '@qYeVniXyC0/D9GIlGMAiIKg5jGgJTY7ZEgeikRWIJ/Y=.ed25519',
+     '@35wu1YDBx9NPsUXpe7bYmHb7BQFEfn2ZFh0DZ6OipA0=.ed25519' ],
+  text: [ 'Hello, can I join?', 'Welcome!' ],
```

Comment:
I may change this text thing to something like: 

```js
comments: [
  { text: 'hello, can I join?' },
  { text: 'welcome! }
]
```

because this is functionally the same but will be able to be upgraded later without breaking the API

Reply:
done